src/ai/backend/agent/agent.py

```diff
@@ -522,6 +522,10 @@ def mount_static_binary(filename: str, target_path: str) -> None:
                 computer_ctx.instance,
                 device_alloc,
             )
+
+            additional_gids = computer_ctx.instance.get_additional_gids()
+            environ["ADDITIONAL_GIDS"] = ",".join(map(str, additional_gids))
+
```

Comment:
This logic will make GIDs only from last accelerator last, not from every accelerators.

Reply:
Thanks for correction!